src/window.js

```diff
@@ -9,15 +9,6 @@ const unlockState = {
     'com.endlessm.hackyballs': [false, false],
 };
 
-function _shouldEnableFlipBack(targetBusName) {
```

Comment:
I assume that we don't need this special case anymore because the framework toolbox now controls whether it's enabled and it only gets enabled for dinosaurs?

Reply:
Correct, and in fact it needs to be disabled initially.

Reply:
We can add it back later if there's a toolbox where the app always needs to be restarted, or just use the toolbox's override of `bindWindow()` to set it.